server/controllers/profile.js

```diff
@@ -51,12 +60,12 @@ exports.newProfile = asyncHandler(async (req, res) => {
   }
 });
 
-// //@route Patch /profile/editprofile/:id
+// //@route Patch /profile/edit-profile/:id
 // //update profiles
 exports.editProfile = asyncHandler(async (req, res) => {
-  const userId = req.params.id;
   const update = req.body;
-
+  let decoded = decodeToken(req.cookies.token);

```

Comment:
We can get the user's id from the request object, we don't need to decode cookies again as it is being done in the auth middleware.
Eg: `req.user.id`

Reply:
I actually got an error when I started my server because this file was missing. I think one of the others created it, my console just told me it was missing so I copied it over from main to correct the error.